docs/dsp-tools-create-ontologies.md

```diff
@@ -907,3 +896,44 @@ it is necessary to reference entities that are defined elsewhere. The following
   These will be created in the exact order they appear in the `ontologies` array. Once an ontology has been created,
   it can be referenced by the following ontologies by its name, e.g. `first-onto:hasName`. It is not necessary to add 
   `first-onto` to the prefixes.
+
+
+## DSP base resources / base properties to be used directly in the XML file
+There is a number of DSP base resources that must not be subclassed in a project ontology. They are directly available 
+in the XML data file:
+
+- `Annotation` is an annotation to another resource of any class. It can be used in the XML file with the 
```

Comment:
is it "annotation to ..." or "annotation of ..."? "to" sounds weird to me but I'm not sure